### PR TITLE
In test - FaF support request start page

### PR DESCRIPTION
**Pull request by PiperS52**
## Changes in this PR

- This is to augment the tests in start_page_spec

### CODE REVIEW

File: spec/features/self-serve/faf/start_page_spec.rb [hunk 1]
```diff
@@ -6,6 +6,23 @@
...
       expect(find("h1")).to have_text "Request help and support with your procurement"
+      expect(page).to have_current_path "/procurement-support"
+    end
+
+    it "navigates to the about frameworks page" do
+      expect(page).to have_link "about frameworks", href: "https://www.gov.uk/guidance/buying-procedures-and-procurement-law-for-schools/find-the-right-way-to-buy", class: "govuk-link"
+    end
+
+    it "navigates to the planning for what you're buying page" do
+      expect(page).to have_link "planning for what you're buying", href: "https://buy-for-your-school-staging.herokuapp.com/beta/phase-6/gov/buying-for-schools", class: "govuk-link"
```
Comment (peterdavidhamilton) [inline spec/features/self-serve/faf/start_page_spec.rb:17]: @shafiqahmedkhan @PiperS52 we don't want links to the prototype in our code, any that have slipped through are an oversight and there is a really like hiding somewhere. Anthony can clarify that for you.

File: spec/features/self-serve/faf/start_page_spec.rb [hunk 2]
```diff
@@ -6,6 +6,23 @@
...
+    end
+
+    it "navigates to the finding the right way to buy page" do
+      expect(page).to have_link "finding the right way to buy", href: "https://buy-for-your-school-staging.herokuapp.com/beta/phase-6/gov/buying-procedures-for-schools", class: "govuk-link"
+    end
+
+    it "navigates to the create a specification page" do
+      expect(page).to have_link "create a specification", href: "https://get-help-buying-for-schools.education.gov.uk/", class: "govuk-link"
```
Comment (peterdavidhamilton) [inline spec/features/self-serve/faf/start_page_spec.rb:25]: @shafiqahmedkhan @PiperS52 this is our specify internal page, but in prod we are now on `service.gov.uk`. The link should be `/dashboard` but confirm with Anthony.

**Review by peterdavidhamilton — CHANGES_REQUESTED** [review]
How are we doing with the link changes?

**Review by peterdavidhamilton — CHANGES_REQUESTED** [review]
Make use of the custom matcher `have_link_to_open_in_new_tab`

**Comment by shafiqahmedkhan** [comment]
LGTM

**Pull request merged**